client/Main.css

```diff
@@ -1,21 +1,27 @@
+@import url(http://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,700italic,300,400,700);
+
 html,
```

Comment:
This shows how to apply a global font so it doesn't need repeated for every item:

https://stackoverflow.com/questions/7025756/how-to-apply-global-font-to-whole-html-document